### ISSUE TITLE
Some tracks raise "RuntimeError: Cannot get alternative track"

### ISSUE THREAD

**Issue by bigchipbag** [Footsiefat/zspotify#3]
Try to download a song like "Nat King Cole Trio - Come in Out of the Rain"

Error below - Should probably handle this exception for bulk downloads.

```
###   FOUND SONG: Nat King Cole Trio - Come In Out Of The Rain    ###
Traceback (most recent call last):
  File "zspotify.py", line 436, in <module>
    main()
  File "zspotify.py", line 433, in main
    client()
  File "zspotify.py", line 125, in client
    downloadTrack(song['track']['id'], name + "/")
  File "zspotify.py", line 360, in downloadTrack
    stream = session.content_feeder().load(
  File "/home/ben/.local/lib/python3.8/site-packages/librespot/audio/__init__.py", line 719, in load
    return self.load_track(playable_id, audio_quality_picker, preload,
  File "/home/ben/.local/lib/python3.8/site-packages/librespot/audio/__init__.py", line 771, in load_track
    raise RuntimeError("Cannot get alternative track")
RuntimeError: Cannot get alternative track

```

**Comment by Footsiefat** [comment]
![image](https://user-images.githubusercontent.com/12180913/137250574-ea715ea8-f7d7-4791-801e-2f4c427cede8.png)
![image](https://user-images.githubusercontent.com/12180913/137250745-948e372c-7ac0-4150-b1eb-ffd3bf6c8aa3.png)
Seems to work fine for me.

How as you picking that song to download? Specific ID as cmd arg or a search?
Also are you using my updated version I updated?

**Comment by bigchipbag** [comment]
Both through a large Playlist ~500 songs, and was with the most recent code
as of a couple hours ago. Will check again tomorrow and give a better
report if it persists.

**Comment by Footsiefat** [comment]
Figured it out...

[https://open.spotify.com/track/1eMBli7nMWvMnndlHPX2Z6](https://open.spotify.com/track/1eMBli7nMWvMnndlHPX2Z6)

here is an example url which causes my program to crash, as you can see you cannot play the track because it is "unavailable" 

![image](https://user-images.githubusercontent.com/12180913/137264578-9d5b673a-4372-47a6-881f-d93174373ac0.png)
![image](https://user-images.githubusercontent.com/12180913/137264626-06b2636a-0c19-4d4c-a9e1-39e68530298c.png)


Basically it would return the track ID when I do the api call so I try to download song but it all breaks since it is unavailable. To fix this I added an additional check before trying to download a track

![image](https://user-images.githubusercontent.com/12180913/137264801-c5e7c9c8-33b1-444f-9568-b388c599e953.png)


Should be fixed in the new version, just give it a test for me and lmk if it works or not thanks.
